spec/rester/client/adapters/local_adapter_spec.rb

```diff
@@ -52,19 +81,28 @@ module Client::Adapters
           let(:path) { '/v1/tests' }
 
           context 'without data' do
-            it { is_expected.to eq [201, '{}'] }
+            it { is_expected.to eq [
+              201,
+              { 'X-Rester-Producer-Name' => 'LocalAdapterTestService' },
+              '{}'] }
           end
 
           context 'with data' do
             let(:params) { { d1: 'one', d2: 2, d3: 3.3 } }
-            it { is_expected.to eq [201, '{"d1":"one","d2":2,"d3":3.3}'] }
+            it { is_expected.to eq [
+              201,
+              { 'X-Rester-Producer-Name' => 'LocalAdapterTestService' },
+              '{"d1":"one","d2":2,"d3":3.3}'] }
```

Comment:
Clean up the formatting here. If a test spans multiple lines, use `do`/`end` syntax.